### DISCUSSION TITLE
I'm new to linux. Could I also install it?

### DISCUSSION THREAD

**Discussion by nataliasmir** [LinSoftWin/Photoshop-CC2022-Linux#75] [Q&A] [answered]
Hello, dear friend

thank you for possibility to use photoshop in linux. But I am unfortunately can not understand how install it/

Could you please type comands that i need to type in terminal? I know well only ctrl+c and ctrl+v ((( I am using kubuntu

**Comment by MiMillieuh** [comment, marked as answer]
Hello, 

Well you just need to setup wine and your graphic drivers properly.

If you're using an AMD or Intel card it should already be installed. if you're using an Nvidia card, you might need to install them with additional driver tool.


For WINE assuming you're using the latest Ubuntu (22.04 LTS) : 

You need to add the 32bit architecture in dpkg. using : `sudo dpkg --add-architecture i386 `

Then you need to add the keys because wine isn't an official repo : `sudo mkdir -pm755 /etc/apt/keyrings && sudo wget -O /etc/apt/keyrings/winehq-archive.key https://dl.winehq.org/wine-builds/winehq.key`

Then add the repo : `sudo wget -NP /etc/apt/sources.list.d/ https://dl.winehq.org/wine-builds/ubuntu/dists/jammy/winehq-jammy.sources`

And then install wine-devel (I recommend it over stable to get the latest patches but you can just install the stable if you want) : `sudo apt install --install-recommends winehq-devel`


Once you're done with that you can download the installer : 
[CC2021](https://raw.githubusercontent.com/MiMillieuh/Photoshop-CC2022-Linux/main/scripts/photoshop2021install.sh)
[CC2021 with camera raw](https://raw.githubusercontent.com/MiMillieuh/Photoshop-CC2022-Linux/main/scripts/photoshop2021installcr.sh)
[CC2022 (Not recommended, really unstable)](https://raw.githubusercontent.com/MiMillieuh/Photoshop-CC2022-Linux/main/scripts/photoshop2022install.sh)


then execute is with `sh photoshop2021install.sh path/to/dir`

Replace path/to/dir with the directory where you want it to be installed (you need to have the permission on that folder.) I personally use a folder called .WineApps that's in my home folder.

And then just wait for it to be installed.

Hopefully it helped.

**Comment by nataliasmir** [comment]
I love you! Will try soon. You are my hero! Great thank You:)

ср, 5 окт. 2022 г., 20:59 MiMillieuh ***@***.***>:

**Reply by nataliasmir** [reply]
Everything works! Great thank You!

**Reply by MiMillieuh** [reply]
Nice, Glad it worked, Have fun ^^

**Reply by nataliasmir** [reply]
sorry i do something wrong 

i wrote sh photoshop2021install.sh wineapps and then instalator start downloading something but in the end i have this

`warning: wine cmd.exe /c echo '%AppData%' returned empty string, error message "wine: invalid directory wineapps/Adobe-Photoshop in WINEPREFIX: not an absolute path" 
------------------------------------------------------
wine: invalid directory wineapps/Adobe-Photoshop in WINEPREFIX: not an absolute path
wine: invalid directory wineapps/Adobe-Photoshop in WINEPREFIX: not an absolute path
wine: invalid directory wineapps/Adobe-Photoshop in WINEPREFIX: not an absolute path
wine: invalid directory wineapps/Adobe-Photoshop in WINEPREFIX: not an absolute path
wine: invalid directory wineapps/Adobe-Photoshop in WINEPREFIX: not an absolute path
wine: invalid directory wineapps/Adobe-Photoshop in WINEPREFIX: not an absolute path
wine: invalid directory wineapps/Adobe-Photoshop in WINEPREFIX: not an absolute path
wine: invalid directory wineapps/Adobe-Photoshop in WINEPREFIX: not an absolute path
chmod: cannot access 'install': No such file or directory
wineapps/Adobe-Photoshop: Not a valid wine prefix.
mkdir: cannot create directory ‘wineapps/Adobe-Photoshop/drive_c/Program Files/Adobe’: No such file or directory
mv: cannot move 'Adobe Photoshop 2021' to 'wineapps/Adobe-Photoshop/drive_c/Program Files/Adobe/Adobe Photoshop 2021': No such file or directory
touch: cannot touch 'wineapps/Adobe-Photoshop/drive_c/launcher.sh': No such file or directory
photoshop2021install.sh: 77: cannot create wineapps/Adobe-Photoshop/drive_c/launcher.sh: Directory nonexistent
photoshop2021install.sh: 78: cannot create wineapps/Adobe-Photoshop/drive_c/launcher.sh: Directory nonexistent
photoshop2021install.sh: 79: cannot create wineapps/Adobe-Photoshop/drive_c/launcher.sh: Directory nonexistent
photoshop2021install.sh: 80: cannot create wineapps/Adobe-Photoshop/drive_c/launcher.sh: Directory nonexistent
photoshop2021install.sh: 81: cannot create wineapps/Adobe-Photoshop/drive_c/launcher.sh: Directory nonexistent
photoshop2021install.sh: 82: cannot create wineapps/Adobe-Photoshop/drive_c/launcher.sh: Directory nonexistent
photoshop2021install.sh: 83: cannot create wineapps/Adobe-Photoshop/drive_c/launcher.sh: Directory nonexistent
photoshop2021install.sh: 84: cannot create wineapps/Adobe-Photoshop/drive_c/launcher.sh: Directory nonexistent
chmod: cannot access 'wineapps/Adobe-Photoshop/drive_c/launcher.sh': No such file or directory
wine: invalid directory wineapps/Adobe-Photoshop in WINEPREFIX: not an absolute path
`

**Reply by MiMillieuh** [reply]
That's because you forgot `~/` that will let the script know that it's in the home folder

**Reply by MiMillieuh** [reply]
For this to work you need or the `~/` or an absolute path that looks like this : `/home/mimillie/.WineApps`

**Comment by nataliasmir** [comment]
I am using photoshop on linux ^_^ Thank You
